pkg/store/mockstore/unistore/tikv/mvcc.go

```diff
@@ -1427,6 +1445,22 @@ func (store *MVCCStore) appendScannedLock(locks []*kvrpcpb.LockInfo, it *locksto
 	return locks
 }
 
+// ScanPessimisticLocks returns matching pessimistic locks.
+func (store *MVCCStore) ScanPessimisticLocks(reqCtx *requestCtx, startTS uint64, forUpdateTS uint64) ([]*kvrpcpb.LockInfo, error) {
```

Comment:
Is it better to be a private method, as it's only internally used by `PessimisticRollbackWithScanFirst`?